src/overlays/actors/ovl_Demo_Kankyo/z_demo_kankyo.c

```diff
@@ -461,24 +461,24 @@ void DemoKankyo_Init(Actor* thisx, PlayState* play) {
 
         case DEMO_KANKYO_TYPE_GIANTS:
             this->isSafeToDrawGiants = false;
-            objId = Object_GetSlot(&play->objectCtx, sObjectBubbleIndex);
+            objectSlot = Object_GetSlot(&play->objectCtx, sObjectBubbleId);
             DemoKankyo_SetupAction(this, DemoKakyo_GiantObjectCheck);
             break;
 
         case DEMO_KANKYO_TYPE_MOON:
-            objId = OBJECT_UNSET_0;
+            objectSlot = OBJECT_UNSET_0;
             this->isSafeToDrawGiants = true;
             DemoKankyo_SetupAction(this, DemoKakyo_MoonSparklesActionFunc);
             break;
 
         default:
             //! @bug: this causes a crash because the actionfunc is never set
-            objId = -1;
+            objectSlot = -1;
             break;
     }
 
-    if (objId > -1) {
-        this->objectId = objId;
+    if (objectSlot > -1) {
```

Comment:
```suggestion
    if (objectSlot >= 0) {
```
Very nitpicky but every other use seems to compare to 0. Would make things easier to macro later if we choose to if they are all the same.

Reply:
Since the `NONE` macro would be `-1`, I think it would be better to change all of them to `-1`. I'll go ahead and just add the macro `OBJECT_SLOT_NONE`?

Reply:
Sure if you want to. Seems like a good idea to me.